### PR TITLE
Prevents the nukeops from declaring war 200 times.

### PR DESCRIPTION
**Pull request by TDcoolguy300**
FIXES THE FUCK OUT OF #2092

### CODE REVIEW

**Comment by Nokington** [comment]
:+1:

**Comment by Pyko1** [comment]
code looks fine, merging since this is a bug fix

File: code/game/gamemodes/nuclear/nuclear_challenge.dm [hunk 1]
```diff
@@ -25,11 +26,19 @@
 		user << "It's too late to declare hostilities. Your benefactors are already busy with other schemes. You'll have to make  do with what you have on hand."
 		return
 
+	if(used) //First used check
```
Comment (Carbonhell) [inline code/game/gamemodes/nuclear/nuclear_challenge.dm:29]: pointless check imo, the one down here was enough

Comment (carlarc) [inline code/game/gamemodes/nuclear/nuclear_challenge.dm:29]: we needed to make sure it was FIXED AS FUCK

**Pull request merged**